src/test/java/pages_sample/ShopPage.java

```diff
@@ -16,22 +18,25 @@
         private WebElement submitButton;
         @FindBy(how = How.CLASS_NAME, using = "error") // By.className("error)
         private WebElement errorText;
-        @FindBy(how = How.NAME, using = "title") // By.name("title")
-        private WebElement getTitle;
-
+        @FindBy(how = How.XPATH, using = "//strong[contains(text(),'0123-456-789')]")
+        private WebElement topPhone;
+        @FindBy(how = How.XPATH, using = "//span[contains(text(),'(347) 466-7432')]")
+        private WebElement bottomPhone;
 
         public String getPageUrl() {
             return "http://automationpractice.com/index.php";
         }
 
-        public getPageTitle() {
-            return getTitle();
+        public void enterName(String name) {
+            nameInput.sendKeys(name);
+        }
 
+        public WebElement getTopPhone() {
+            return topPhone;
         }
 
-        public void enterName(String name) {
-            nameInput.clear();
-            nameInput.sendKeys(name);
+        public WebElement getBottomPhone() {
+            return bottomPhone;
```

Comment:
I would recommend make web elements public. And, in case you are doing one action - call this web element from steps class.